pandas/tests/extension/base/methods.py

```diff
@@ -126,11 +127,26 @@ def test_combine_add(self, data_repeated):
         orig_data1, orig_data2 = data_repeated(2)
         s1 = pd.Series(orig_data1)
         s2 = pd.Series(orig_data2)
-        result = s1.combine(s2, lambda x1, x2: x1 + x2)
+
+        # fundamentally this is not a great operation
```

Comment:
FYI @Dr-Irv

Reply:
@jreback Not sure what you mean.  I added the test to see that `Series.combine()` works for EA.

Reply:
yes i may do that